updater/lib/dependabot/dependency_change.rb

```diff
@@ -3,6 +3,7 @@
 
 require "sorbet-runtime"
 require "dependabot/errors"
+require "dependabot/pull_request_creator/message_builder"
```

Comment:
🔎 This doesn't seem to map to (new) usage within the file. Was this missing, or is this a remnant of development that can go away?

Reply:
The `PullRequestCreator` is used at https://github.com/dependabot/dependabot-core/blob/471d5a548dac32cd475a5b257b6ab87e8e9bb7bb/updater/lib/dependabot/dependency_change.rb#L62 and if you try to run the `dependency_change_spec` tests without the require then it will raise a class not found error.

I think it's incidentally being required when rubocop is run from the root directory, so I'm making the requirement explicit here.

Reply:
Thanks for the context!